### PR TITLE
Br add experimental description for acceleration restore speed

### PR DESCRIPTION
**Pull request by benmaoer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/br-snapshot-guide.md [hunk 1]
```diff
@@ -71,9 +71,11 @@ tiup br restore full --pd "${PD_IP}:2379" \
 --storage "s3://backup-101/snapshot-202209081330?access-key=${access-key}&secret-access-key=${secret-access-key}"
 ```
 
-为了解决大规模 Region 场景下，打散 Region 可能成为恢复过程中的瓶颈问题，BR 从 v7.6.0 开始支持以粗粒度打散 Region 的方式进行恢复（实验特性）。你可以通过指定 `--granularity="coarse-grained"` 启用该方式。
+> **注意：**
+>
+> - 在 TiDB v7.6.0 版本中，通过设置 `--granularity="coarse-grained"` 参数启用粗粒度打散 Region 的方式加速恢复是实验性功能，建议在低于 1,000 张表的集群中使用此功能来加速数据恢复。请注意，该功能还未支持断点恢复。
```
Comment (Oreoxmt) [inline br/br-snapshot-guide.md:76]:
```suggestion
> **警告：**
>
> 在 v7.6.0 中，通过设置 `--granularity="coarse-grained"` 参数启用粗粒度打散 Region 算法加速恢复为实验特性，建议在表数量不超过 1,000 的集群中使用此功能加速数据恢复。请注意，该功能暂不支持断点恢复。
```

File: br/br-snapshot-guide.md [hunk 2]
```diff
@@ -71,9 +71,11 @@ tiup br restore full --pd "${PD_IP}:2379" \
...
 
-在恢复快照备份数据过程中，终端会显示恢复进度条。在完成恢复后，会输出恢复耗时、速度、恢复数据大小等信息。
+为了更进一步加速大集群的恢复速度，BR 从 v7.6.0 开始支持通过设置 `--granularity="coarse-grained"` 参数启用粗粒度打散 Region 的方式（实验特性）来进行更快的并行恢复。通过优化 Region 打散算法，BR 可以将恢复任务快速拆分为多个小任务，并批量分散到所有 TiKV 节点上充分利用每个 TiKV 节点的资源，实现了更快的并行恢复速度。
```
Comment (Oreoxmt) [inline br/br-snapshot-guide.md:78]:
```suggestion
为进一步提升大集群的恢复速度，BR 从 v7.6.0 开始支持通过设置 `--granularity="coarse-grained"` 参数启用粗粒度打散 Region 算法（实验特性）进行更快的并行恢复。通过优化 Region 打散算法，BR 可以迅速将恢复任务拆分为大量小任务，并批量分散到所有 TiKV 节点上。粗粒度打散 Region 算法充分利用每个 TiKV 节点的所有资源，实现了并行快速恢复。
```

**Review by Leavrth — APPROVED** [review]

File: br/br-snapshot-guide.md (hunk 2 above)
Comment (Leavrth) [reply br/br-snapshot-guide.md:78]: 粗粒度打算是加快 split & scatter region 的速度。

**Review by Oreoxmt — APPROVED** [review]

File: br/br-snapshot-guide.md (hunk 2 above)
Comment (benmaoer) [reply br/br-snapshot-guide.md]: 是的，整体上也加速了恢复

**Review by qiancai — APPROVED** [review]

File: br/br-snapshot-guide.md [hunk 3]
```diff
@@ -71,7 +71,11 @@ tiup br restore full --pd "${PD_IP}:2379" \
 --storage "s3://backup-101/snapshot-202209081330?access-key=${access-key}&secret-access-key=${secret-access-key}"
 ```
 
-为了解决大规模 Region 场景下，打散 Region 可能成为恢复过程中的瓶颈问题，BR 从 v7.6.0 开始支持以粗粒度打散 Region 的方式进行恢复（实验特性）。你可以通过指定 `--granularity="coarse-grained"` 启用该方式。
+> **警告：**
+>
+> 在 v7.6.0 中，通过设置 `--granularity="coarse-grained"` 参数启用粗粒度打散 Region 算法加速恢复为实验特性，建议在表数量不超过 1,000 的集群中使用此功能加速数据恢复。请注意，该功能暂不支持断点恢复。
```
Comment (qiancai) [inline br/br-snapshot-guide.md:76]:
```suggestion
> 通过设置 `--granularity="coarse-grained"` 参数启用粗粒度打散 Region 算法加速恢复为实验特性，建议在表数量不超过 1,000 的集群中使用此功能加速数据恢复。请注意，该功能暂不支持断点恢复。
```

Comment (qiancai) [reply br/br-snapshot-guide.md:76]: reason:  v8.0 这个特性可能还是实验特性

File: br/br-snapshot-guide.md [hunk 4]
```diff
@@ -71,7 +71,11 @@ tiup br restore full --pd "${PD_IP}:2379" \
...
+
+为进一步提升大集群的恢复速度，BR 从 v7.6.0 开始支持通过设置 `--granularity="coarse-grained"` 参数启用粗粒度打散 Region 算法（实验特性）进行更快的并行恢复。通过优化 Region 打散算法，BR 可以迅速将恢复任务拆分为大量小任务，并批量分散到所有 TiKV 节点上。粗粒度打散 Region 算法充分利用每个 TiKV 节点的所有资源，实现了并行快速恢复。
```
Comment (qiancai) [inline br/br-snapshot-guide.md:78]:
```suggestion
为进一步提升大集群的恢复速度，BR 从 v7.6.0 开始支持通过设置 `--granularity="coarse-grained"` 参数启用粗粒度打散 Region 算法（实验特性）进行更快的并行恢复。启用后，BR 可以迅速将恢复任务拆分为大量小任务，并批量分散到所有 TiKV 节点上，从而充分利用每个 TiKV 节点的所有资源，实现并行快速恢复。
```

**Review by BornChanger — APPROVED** [review]

**Pull request merged**